typescript-operators/main.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+
+interface Customer {
+  name: string;
+  details?: {
+    address?: {
+      city: string;
+      street: string;
+      zip: string;
+    };
+    age?: number;
+  };
+}
+
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+const v5: Customer = {
+  name: 'Carl',
+  details: {
+    age: 82,
+  },
+};
+
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```

Comment:
Not quite! This is ALWAYS going to be logging something. If you check your screenshot, you can see you are getting an empty console log. The `console.log` should only fire if the first operand is truthy.